docs/USAGE.rst

```diff
@@ -0,0 +1,366 @@
+
+When an image is first imported into the WSI DeID from the host filesystem, it will be renamed according to the import process and will be in the ``Available To Process`` folder.
+
+Step 3 Users Process and Deidentify Files
+-----------------------------------------
+
+Once an image is in the ``Available To Process`` folder, the user can make the following UI selections:
+
+- "Redact Image" to redact it, sending it to the ``Redacted`` folder, and sending an original file to the ``Original`` folder (go to step 4)
+- "Quarantine" for more reprocessing, sending it to the ``Quarantine`` folder (go to step 4a)
+- "Reject" to mark that it is impossible to fix, sending it to the ``Rejected`` folder (go to Step 4b)
+
+An image is copied into the ``Original`` folder before it will be redacted and will go into the ``Redacted`` folder, so that an unedited copy of the image is stored.
+
+Step 4 Users Approve Deidentified Files
+---------------------------------------
+
+Images in the ``Redacted`` folder have gone through the redaction process, but should be inspected to determine if they still contain PHI/PII or are fully cleared and ready for release.
+
+Once an image is in the ``Redacted`` folder, the user can approve the file. The user can make the following UI selections:
+
+- "Approve" to approve it, once it has been fully cleared for release. If this is pressed, then the image will move to the ``Approved`` folder and then the view will change to the next image to be processed, as if you had clicked on the ``Next Item`` action in the left menu. 
+- "Quarantine" for more reprocessing, sending it to the ``Quarantine`` folder (go to step 4a)
+
+Step 4a Users Contact IMS and Reprocess for Potential Resolution
+----------------------------------------------------------------
+
+The ``Quarantine`` folder is for holding images that may hold PHI/PII. Users should contact IMS for any WSI files in the ``Quarantine`` folder, and then review files for reprocessing based on instructions from IMS. If PHI/PII is seen in an image or metadata field, that is somewhere other than the ``Available To Process`` folder, it should be quarantined for reprocessing.
+
+Step 4b Users Review and Re-process Rejected Files After Software Updates
+-------------------------------------------------------------------------
+
+The ``Rejected`` folder  is available at any time.
+
+If an image is determined to be impossible to fix--perhaps it is too difficult to confirm that PHI/PII has been removed, or if so much data would be removed to de-identify the image that the image data would be useless for research purposes--then the image can be sent to the ``Rejected`` folder by clicking on the ``rejected`` button. Users should contact IMS for any WSI files in the ``Rejected`` folder.
+
+TODO: provide specific examples of when an image should be rejected.
```

Comment:
There are two ways an image needs to be rejected: (a) there is PHI in the zoomable image in the WSI (e.g., someone wrote the patient name on the slide itself), (b) there is PHI in a metadata field that we show but do not offer a redaction control for (I cannot think how this could actually occur, but the human review is the fall-back for that belief).